src/app/[locale]/(main)/imprint/page.tsx

```diff
@@ -0,0 +1,62 @@
+import '@/themes/theme.karlsruhe.scss';
+
+import { useTranslations } from 'next-intl';
+
+export default function Imprint() {
+  const t = useTranslations('Footer');
+  return (
+    <div className="container mt-4">
+      <h1 className="custom-heading">{t('imprint')}</h1>
+      <br />
+      <div className="mb-4">
+        <h2 className="custom-heading">{t('publisher')}</h2>
+        <address>
+          Stadt Karlsruhe <br />
+          Karl-Friedrich-Str. 10 <br />
+          76133 Karlsruhe <br />
+          Telefon: 0721 133-0 <br />
+          E-Mail: <a href="mailto:stadt@karlsruhe.de">stadt[at]karlsruhe.de</a>
+        </address>
+        <p>
+          <b>{t('authorizedRepresentative')}</b> <br />
+          Oberbürgermeister Dr. Frank Mentrup
+        </p>
+        <p>
+          <b>{t('supervisoryAuthority')}</b> <br />
+          Regierungspräsidium Karlsruhe
+        </p>
+        <p>
+          <b>{t('vatId')}</b> <br />
+          DE143589000
+        </p>
+        <p>
+          <b>{t('postalAddress')}</b> <br />
+          Stadt Karlsruhe <br />
+          76124 Karlsruhe
+        </p>
+      </div>
+
+      <div className="mb-4">
+        <h3 className="custom-heading">{t('responsibleEditorialDepartment')}</h3>
+        <p>
+          Amt für Informationstechnik und Digitalisierung <br />
+          Open Government und Open Data <br />
+          E-Mail: <a href="mailto:transparenz@karlsruhe.de">transparenz[at]karlsruhe.de</a>
+        </p>
+      </div>
+
+      <div className="mb-4">
+        <h3 className="custom-heading">{t('technicalImplementation')}</h3>
+        <p>Anwendungsprojekt Hochschule Karlsruhe</p>
```

Comment:
We could use our actual names here (if desired, with GitHub and LinkedIn links even)

Reply:
Can be handled in a follow-up ticket